packages/manager/apps/pci-load-balancer/src/components/listing/ProvisioningStatus.component.tsx

```diff
@@ -0,0 +1,46 @@
+import { useTranslation } from 'react-i18next';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
+import { OdsChipAttribute } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+type ProvisioningStatusComponentProps = {
+  provisioningStatus: string;
+};
+
+export default function ProvisioningStatusComponent({
+  provisioningStatus,
+}: Readonly<ProvisioningStatusComponentProps>) {
+  const { t } = useTranslation('');
+  const [chipAttribute, setChipAttribute] = useState<OdsChipAttribute>({
+    color: ODS_THEME_COLOR_INTENT.success,
+  });
+  useEffect(() => {
```

Comment:
when you have useState + useEffect most of the time you can replace it with useMemo

```
const chipAttribute = useMemo(() => ..., [provisioningStatus]);
```